ext/pdo/pdo_sql_parser.re

```diff
@@ -394,6 +430,7 @@ rewrite:
 			efree(name);
 			plc->quoted = "?";
 			plc->qlen = 1;
+			newbuffer_len -= plc->len - 1;
```

Comment:
The changes to the buffer length calculation look a bit scary ... can there be any double subtraction here, once from `- escapes` where it is used, and once from len 2 to qlen 1 here?

Reply:
It seems to me that they are two different code paths.

Reply:
You were right, good catch!